### ISSUE TITLE
the default example cause error

### ISSUE THREAD

**Issue by leoliuzcl** [apache/rocketmq-client-go#150]
container list:

```
CONTAINER ID        IMAGE                               COMMAND             CREATED             STATUS              PORTS                                                                     NAMES
ed9af94d9ede        rocketmqinc/rocketmq:4.5.0-alpine   "sh mqbroker"       2 days ago          Up 24 minutes       0.0.0.0:10909->10909/tcp, 9876/tcp, 10912/tcp, 0.0.0.0:10911->10911/tcp   rmqbroker
4f082d648ebc        rocketmqinc/rocketmq:4.5.0-alpine   "sh mqnamesrv"      2 days ago          Up 24 minutes       10909/tcp, 0.0.0.0:9876->9876/tcp, 10911-10912/tcp                        rmqnamesrv
```

docker exec -it ed9af94d9ede  ./mqadmin clusterList -n namesrv:9876

```
OpenJDK 64-Bit Server VM warning: ignoring option PermSize=128m; support was removed in 8.0
OpenJDK 64-Bit Server VM warning: ignoring option MaxPermSize=128m; support was removed in 8.0
#Cluster Name     #Broker Name            #BID  #Addr                  #Version                #InTPS(LOAD)       #OutTPS(LOAD) #PCWait(ms) #Hour #SPACE
DefaultCluster    ed9af94d9ede            0     172.17.0.3:10911       V4_5_0                   0.00(0,0ms)         0.00(0,0ms)          0 434920.32 -1.0000
```

topicList with cmd: docker exec -it ed9af94d9ede  ./mqadmin  topicList

```
docker exec -it ed9af94d9ede  ./mqadmin  topicList
OpenJDK 64-Bit Server VM warning: ignoring option PermSize=128m; support was removed in 8.0
OpenJDK 64-Bit Server VM warning: ignoring option MaxPermSize=128m; support was removed in 8.0
sayliu
RMQ_SYS_TRANS_HALF_TOPIC
BenchmarkTest
OFFSET_MOVED_EVENT
TBW102
SELF_TEST_TOPIC
ed9af94d9ede
DefaultCluster
```

cmd with the example code
```
go run main.go produce -t sayliu -b {"b":123}
```

error:
```
config ProducerConfig=[GroupId: testGroup, NameServer: localhost:9876, LogConfig: {Path:example FileNum:1 FileSize:65536 Level:Trace}, ]
create Producer succ, error: <nil>
Producer: ProducerConfig=[GroupId: testGroup, NameServer: localhost:9876, LogConfig: {Path:example FileNum:1 FileSize:65536 Level:Trace}, ] started...
WARN[0012] send message error, error is: send message with sync failed
Error: send message with sync failed
send message: {b:123}-0 result: <nil>
```

can somebody help?

**Comment by wenfengwang** [comment]
It seems that you go client version is cgo, could you instead it with the native? 2.0.0-alpha2 is available now, and the 2.0.0-rc1 will available soon

**Comment by leoliuzcl** [comment]
> It seems that you go client version is cgo, could you instead it with the native? 2.0.0-alpha2 is available now, and the 2.0.0-rc1 will available soon
native worked! thank you
